worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -293,6 +293,9 @@ private[celeborn] class Worker(
 
   private val masterClient = new MasterClient(internalRpcEnvInUse, conf, true)
 
+  private val rackResolver = new CelebornRackResolver(conf)
+  private val networkLocation = rackResolver.resolve(host).getNetworkLocation
```

Comment:
Are we making the assumption here that the `DNSToSwitchMapping` implementation is present on all the Worker nodes? Earlier the assumption was that this was only present on the Master node. In existing production environment, if this is not present on the worker nodes, this will break the replication across racks feature.

Reply:
CelebornRackResolver specifies `ScriptBasedMapping` as a default value that is returned ([code](https://github.com/apache/incubator-celeborn/blob/main/master/src/main/scala/org/apache/celeborn/service/deploy/master/network/CelebornRackResolver.scala#L47)), so I don't think it would throw an exception if an impl is not provided by the user.